CHI_TECH/ChiMesh/SweepUtilities/chi_angleaggregation.h

```diff
@@ -29,10 +29,11 @@
  * The octant based seperation is achieved via the notion of "AngleSetGroup"
  * which will of a group of angle sets for each quadrant or octant
```

Comment:
Not related to the change you made, but there are just too many typos in the three context lines that Github shows to avoid my attention 😄 

```
seperation  -->  separation
which will of a group of angle sets  -->  which will group angle sets
depenending  -->  depending
```

Reply:
👍 Fixed.